datasets/wiki-text/metadata.json

```diff
@@ -0,0 +1,121 @@
+  "license": "https://creativecommons.org/licenses/by-sa/3.0/",
+  "citation": "@article{merity2016pointer, title={Pointer sentinel mixture models}, author={Merity, Stephen and Xiong, Caiming and Bradbury, James and Socher, Richard}, journal={arXiv preprint arXiv:1609.07843}, year={2016} }",
+  "distribution": [
+    {
+      "name": "wikitext-103-v1.zip",
+      "@type": "sc:FileObject",
+      "contentUrl": "https://s3.amazonaws.com/research.metamind.io/wikitext/wikitext-103-v1.zip",
+      "sha256": "242ba0f20b329cfdf1ccc61e9e9e5b59becf189db7f7a81cd2a0e2fc31539590",
+      "encodingFormat": "application/zip"
+    },
+    {
+      "name": "token-files",
+      "@type": "sc:FileSet",
+      "containedIn": [
+        "#{wikitext-103-v1.zip}"
+      ],
+      "includes": "wikitext-103/*.tokens",
+      "encodingFormat": "image/jpeg"
+    },
+    {
+      "@type": "sc:FileObject",
+      "name": "splits.csv",
+      "contentUrl": "data/splits.csv",
+      "sha256": "dec55b82438aba979670b0eabd99b1d8cc9d385cef028314feccba1cc68af33b",
+      "encodingFormat": "text/csv"
+    }
+  ],
+  "recordSet": [
+    {
+      "@type": "ml:RecordSet",
+      "name": "split_enums",
+      "description": "Maps split names to semantic values.",
+      "source": "#{splits.csv}",
+      "key": "#{name}",
+      "field": [
+        {
+          "name": "name",
+          "description": "One of: train, valid, test.",
+          "@type": "ml:Field",
+          "source": "#{splits.csv/name}",
+          "dataType": "sc:Text"
+        },
+        {
+          "name": "url",
+          "description": "Corresponding mlcommons.org definition URL",
+          "@type": "ml:Field",
+          "source": "#{splits.csv/url}",
+          "dataType": [
+            "sc:Url",
+            "wd:Q3985153"
+          ]
+        }
+      ]
+    },
+    {
+      "@type": "ml:RecordSet",
+      "name": "words",
+      "description": "The words of the token files, including their split.",
+      "source": "#{token-files}",
+      "field": [
+        {
+          "name": "word",
+          "@type": "ml:Field",
+          "description": "A word.",
+          "dataType": "sc:Text",
+          "repeated": "true",
+          "source": {
+            "data": "#{token-files/content}",
+            "applyTransform": [
+              {
+                "replace": "\\n/<eos>"
```

Comment:
Note for the reviewers:

1. applyTransform doesn't allow for lists in the current specs. Should it? Or can I do this more elegantly?
2. replace is not part of the current [specs](https://docs.google.com/document/d/1EDUbwwSeSUpRZWczy6FNuCXTbKh9e5H4xXMnqgoCRj8/edit#bookmark=id.z90k3xy7xz3q). Should it be?

Reply:
1. Yes I think it might make sense to support list, which unlike dicts are sorted.
2. Replace sounds to me like something we would want to support. Although we probably don't want to support arbitrary transformations, this one sounds simple enough and common too.